From b9a88896d07cb8ec864a71f936da52137b8fd533 Mon Sep 17 00:00:00 2001
From: Tamas Szucs <tamas.szucs@u-blox.com>
Date: Tue, 12 Mar 2019 16:21:17 +0100
Subject: [PATCH] switch from vfs_write() to kernel_write()

Signed-off-by: Tamas Szucs <tamas.szucs@u-blox.com>
---
 dhd_linux.c | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/dhd_linux.c b/dhd_linux.c
index 481fd3d..6eaaa93 100644
--- a/dhd_linux.c
+++ b/dhd_linux.c
@@ -12133,7 +12133,11 @@ write_to_file(dhd_pub_t *dhd, uint8 *buf, int size)
 	}
 
 	/* Write buf to file */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14))
 	ret = vfs_write(fp, buf, size, &pos);
+#else
+	ret = kernel_write(fp, buf, size, &pos);
+#endif /* (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14)) */
 	if (ret < 0) {
 		DHD_ERROR(("write file error, err = %d\n", ret));
 		goto exit;
@@ -13765,7 +13769,11 @@ do_dhd_log_dump(dhd_pub_t *dhdp)
 		goto exit;
 	}
 
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14))
 	ret = vfs_write(fp, common_info, strlen(common_info), &pos);
+#else
+	ret = kernel_write(fp, common_info, strlen(common_info), &pos);
+#endif /* (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14)) */
 	if (ret < 0) {
 		DHD_ERROR(("write file error, err = %d\n", ret));
 		goto exit;
@@ -13776,7 +13784,11 @@ do_dhd_log_dump(dhd_pub_t *dhdp)
 		size = (unsigned int)(dhdp->dld_buf.present - dhdp->dld_buf.front);
 	}
 
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14))
 	ret = vfs_write(fp, dhdp->dld_buf.buffer, size, &pos);
+#else
+	ret = kernel_write(fp, dhdp->dld_buf.buffer, size, &pos);
+#endif /* (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14)) */
 	if (ret < 0) {
 		DHD_ERROR(("write file error, err = %d\n", ret));
 		goto exit;
@@ -14519,7 +14531,11 @@ dhd_write_file(const char *filepath, char *buf, int buf_len)
 		ret = BCME_ERROR;
 	} else {
 		if (fp->f_mode & FMODE_WRITE) {
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14))
 			ret = vfs_write(fp, buf, buf_len, &fp->f_pos);
+#else
+			ret = kernel_write(fp, buf, buf_len, &fp->f_pos);
+#endif /* (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 14)) */
 			if (ret < 0) {
 				DHD_ERROR(("%s: Couldn't write file '%s'\n",
 					__FUNCTION__, filepath));
-- 
2.11.0

