From 30494ce86bd03c8e92b085f4e2e8b6ac71435cb0 Mon Sep 17 00:00:00 2001
From: Tamas Szucs <tamas.szucs@u-blox.com>
Date: Thu, 28 Mar 2019 13:07:03 +0100
Subject: [PATCH 2/2] declare flag EXT_HOST_MLME from linux 3.8.0

Signed-off-by: Tamas Szucs <tamas.szucs@u-blox.com>
---
 wlan_src/mlinux/moal_main.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wlan_src/mlinux/moal_main.h b/wlan_src/mlinux/moal_main.h
index 9913a8f..3beafb3 100755
--- a/wlan_src/mlinux/moal_main.h
+++ b/wlan_src/mlinux/moal_main.h
@@ -1803,7 +1803,7 @@ enum ext_mod_params {
 #endif
 	EXT_ROAMOFFLOAD_IN_HS,
 #ifdef STA_CFG80211
-#if CFG80211_VERSION_CODE >= KERNEL_VERSION(3, 11, 0)
+#if CFG80211_VERSION_CODE >= KERNEL_VERSION(3, 8, 0)
 	EXT_HOST_MLME,
 #endif
 #endif
-- 
2.11.0

